src/GraphQL.Conventions/Web/RequestHandler.cs

```diff
@@ -35,12 +34,12 @@ public class RequestHandlerBuilder : IDependencyInjector
 
             internal RequestHandlerBuilder()
             {
-                _dependencyInjector = this;
+                _serviceProvider = this;
```

Comment:
Note: see constructor on line 46: `WithDependencyInjector(ResolveTypeDelegate resolveTypeDelegate)`

Shouldn't that constructor also contain `_serviceProvider = this;` ??